### PR TITLE
Swolfson-Project-express-api

### PR DESCRIPTION
**Pull request by swolfson1**

### CODE REVIEW

**Comment by JennieDalgren** [comment]
add link to deployed project please

**Comment by swolfson1** [comment]
> add link to deployed project please

https://polar-anchorage-36924.herokuapp.com/

File: server.js [hunk 1]
```diff
@@ -1,33 +1,63 @@
...
 // Start defining your routes here
-app.get('/', (req, res) => {
-  res.send('Hello world')
-})
+app.get("/", (req, res) => {
+  res.send("Hello world");
+});
+app.get("/nominations", (req, res) => {
+  res.json(goldenGlobesData);
+  res.send("happy");
```
Comment (JennieDalgren) [inline server.js]: remove this line

File: server.js [hunk 2]
```diff
@@ -1,33 +1,63 @@
...
+});
+
+app.get("/year/:year", (req, res) => {
```
Comment (JennieDalgren) [inline server.js]: ⭐️

**Pull request closed without merging**